src/app/pages/About/About.html

```diff
@@ -0,0 +1,18 @@
+<div class="about">
+  <h1>{{ $t('app.title') }}</h1>
+
+  <h2>Contributors</h2>
+
+  <div class="row justify-content-center">
+    <div class="col-lg-2 col-md-4 col-sm-6 col-xs-12">
```

Comment:
![vue](https://user-images.githubusercontent.com/5292838/39778562-f2af7748-5306-11e8-8e54-7a21c8175842.gif)
Jump between small and medium devices is too high, and on small devices, there is no margin

Reply:
No need as it will be refactored anyway to meet our boilerplate example page.